src/autocomplete/PeopleAutocomplete.js

```diff
@@ -74,21 +90,28 @@ export default function PeopleAutocomplete(props: Props): Node {
         />
       ),
     }: Section<UserGroup>),
+    ({
+      data: wildcardMentionForQuery != null ? [wildcardMentionForQuery] : [],
```

Comment:
It looks like we're not actually using in this component the fact that there's just one of these, right?

In that case, likely simpler to have the helper function return the list, and have it just be an internal fact about that function that it only ever returns at most one item.